packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -157,57 +234,54 @@ export function applyFieldEdit(tree: FuzzView, fieldEdit: FieldEdit): void {
 
 function applySequenceFieldEdit(
 	tree: FuzzView,
-	field: FlexTreeSequenceField<readonly [Any]>,
+	parentNode: FuzzNode,
 	change: Insert | Remove | IntraFieldMove | CrossFieldMove,
 ): void {
 	switch (change.type) {
 		case "insert": {
-			field.editor.insert(
-				change.index,
-				mapTreeFieldFromCursor(cursorForJsonableTreeField(change.content)),
+			const insertValues = change.content.map((value) =>
+				generateFuzzLeafNode(value, tree.currentSchema),
 			);
+			for (let index = change.index; index < insertValues.length; index++) {
+				parentNode.arrayChildren.insertAt(index, insertValues[index - change.index]);
+			}
```

Comment:
This is inserting the nodes one by one, which is not a interesting for testing purposes as inserting them in one block. You may be able to use `insertAt(index, TreeArrayNode.spread(insertValues))` to do that.

Reply:
Updated to insert values in one block